engine/src/main/java/org/terasology/rendering/dag/nodes/BlurNode.java

```diff
@@ -79,7 +76,7 @@ protected void setupConditions(Context context) { }
      */
     @Override
     public void process() {
-        PerformanceMonitor.startActivity("rendering/" + label);
+        PerformanceMonitor.startActivity("rendering/" + getUri());
```

Comment:
It would certainly be possible - and recommended. Would you like me to it in this PR, or a new one?

Reply:
My suspicion is that it wouldn't require a whole lot of code changes. Try and do it. If you find unforseen difficulties let's leave it for a separate PR. If it is as easy as I think it is, let's do it in this PR.

Reply:
Beep Boop, unforseen difficulties detected!

While iterating over `renderPipelineTaskList`, every object is a `RenderPipelineTask`. We added nodeUri to `Nodes`, not to `RenderPipelineTasks`. This will be slightly tricky. If you don't mind, I'm deferring this for a later PR.

Reply:
Ok.